packages/laravel/src/HybridlyServiceProvider.php

```diff
@@ -80,6 +82,18 @@ public function packageBooted(): void
         }
     }
 
+    protected function registerOctaneListener(): void
+    {
+        if (!class_exists(RequestReceived::class)) {
+            return;
+        }
+
+        $this->app['events']->listen(
+            RequestReceived::class,
```

Comment:
Think you'd also need to do this for `TaskReceived::class` and `TickReceived::class`

Reply:
Thanks!